ocp_resources/backup.py

```diff
@@ -11,17 +11,29 @@ class Backup(NamespacedResource):
 
     api_group = NamespacedResource.ApiGroup.VELERO_IO
 
-    def __init__(self, included_namespaces=None, excluded_resources=None, **kwargs):
+    def __init__(
+        self,
+        included_namespaces=None,
+        excluded_resources=None,
+        snapshot_move_data=False,
```

Comment:
please update the API url to refer to a version that contains the new attributes; better refer to main

Reply:
I added the doc about backup with the datamover

Reply:
this is the link that should be used
https://velero.io/docs/main/api-types/backup/ in the class docstring

Reply:
Done

Reply:
which replaces
https://github.com/RedHatQE/openshift-python-wrapper/pull/1541/files#diff-ce8d6a2664a6fcef39b9eee8abb5db640a59cc1dbf7b889c7392d4f95ce73bf7R7-R10